extra/resources/CPUIOWaitHealth

```diff
@@ -0,0 +1,192 @@
+#!/bin/sh
+#
+# CPUIdle RA
+# Measures CPU iowait % via iostat and writes #health-cpu-iowait status into the CIB
+#
+# Originally based on ocf:pacemaker:HealthCPU RA
+# Copyright 2004-2009 SUSE LINUX AG, Lars Marowsky-Brée. All Rights Reserved.
+#
+# Later changes copyright 2009-2018 Michael Schwartzkopff
+# in collaboration with the Bull company. Merci! All Rights Reserved.
+#
+# Later changes copyright 2018 Maciej Sobkowiak <asm@egnyte.com>
+#
+# This source code is licensed under the GNU General Public License version 2
+# (GPLv2) WITHOUT ANY WARRANTY.
+#
+#######################################################################
+# Initialization:
+
+: ${OCF_FUNCTIONS=${OCF_ROOT}/resource.d/heartbeat/.ocf-shellfuncs}
+. ${OCF_FUNCTIONS}
+: ${__OCF_ACTION=$1}
+
+#######################################################################
+
+meta_data() {
+	cat <<END
+<?xml version="1.0"?>
+<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
+<resource-agent name="CPUIdle" version="1.0">
+<version>1.0</version>
+
+<longdesc lang="en">
+Systhem health agent that measures the CPU iowait via iostat and updates the #health-cpu-iowait attribute.
+</longdesc>
+<shortdesc lang="en">System health CPU iowait measurement</shortdesc>
+
+<parameters>
+<parameter name="state" unique="1">
+<longdesc lang="en">
+Location to store the resource state in.
+</longdesc>
+<shortdesc lang="en">State file</shortdesc>
+<content type="string" default="${HA_VARRUN%%/}/health-cpu-iowait-${OCF_RESOURCE_INSTANCE}.state" />
+</parameter>
+
+<parameter name="yellow_limit" unique="1">
+<longdesc lang="en">
+Upper (!) limit of iowait percentage to switch the health attribute to yellow. I.e.
+the #health-cpu-iowait will go yellow if the %iowait of the CPU gets higher than 10%.
+</longdesc>
+<shortdesc lang="en">Upper limit for yellow health attribute</shortdesc>
+<content type="string" default="10"/>
+</parameter>
+
+<parameter name="red_limit" unique="1">
+<longdesc lang="en">
+Upper (!) limit of iowait percentage to switch the health attribute to red. I.e.
+the #health-cpu-iowait will go red if the %iowait of the CPU get higher than 15%.
+</longdesc>
+<shortdesc lang="en">Upper limit for red health attribute</shortdesc>
+<content type="string" default="15"/>
+</parameter>
+
+</parameters>
+
+<actions>
+<action name="start"        timeout="10s" />
+<action name="stop"         timeout="10s" />
+<action name="monitor"      timeout="10s" interval="10s" start-delay="0s" />
+<action name="meta-data"    timeout="5s" />
+<action name="validate-all" timeout="10s" />
+</actions>
+</resource-agent>
+END
+}
+
+#######################################################################
+
+dummy_usage() {
```

Comment:
I know the Dummy agent was the original source for this (via HealthCPU), but it would make sense to use something other than "dummy_" as a prefix. Maybe "agent_".

Reply:
Changed to agent_*